pkg/validation/OWNERS

```diff
@@ -0,0 +1,11 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+options:
+  no_parent_owners: true
+filters:
```

Comment:
filters doesn't work for approvers, should just be 

```
approvers:
...
reviewers:
...
```

Reply:
oh, right. Copy&paste from api repo. But there it was restricted to different file types.